internal/hooks/loghooks.go

```diff
@@ -19,32 +19,6 @@ const StopLogAfter keyType = "stopLogAfter"
 const ThresholdWhenLogMqtt keyType = "threshold"
 const PowerStatus keyType = "powerStatus"
 
-type LogHook struct {
-	DataSender network.DataSender
-}
-
-// Run All fatal log level will be sent to mqtt,
-// if we want error log level be sent to mqtt bus, we need to log it like:
-// logger.Error().Ctx(context.WithValue(context.Background(), internal.LogToMqtt, "true")).Msgf("Error msg: %s", err.Error())
-func (h *LogHook) Run(e *zerolog.Event, level zerolog.Level, msg string) {
-	// This is where you can modify the event by adding fields, changing
-	// existing ones, or even decide to not log the event.
-	if level == zerolog.FatalLevel {
-		err := h.DataSender.SendErrorPayload(errors.New(msg), nil)
-		if err != nil {
-			return
-		}
-	}
-
-	if e.GetCtx().Value(LogToMqtt) != nil {
-		err := h.DataSender.SendErrorPayload(errors.New(msg), nil)
```

Comment:
We had two hooks in loghooks.go, now I deleted one, since I do not think that we need both.
[Here is](https://github.com/DIMO-Network/edge-network/blob/776b5af2b1f130ac8ed23a13a189d82272baaf57/internal/hooks/loghooks.go#L71-L70) how we send the data over mqtt using datasender in LogRateLimiterHook